lib/stream/leo-stream.js

```diff
@@ -765,7 +765,7 @@ module.exports = function(configure) {
 					logger.time("firehose request");
 					firehose.putRecordBatch({
 						Records: [{
-							Data: records.join('')
+							Data: opts.gzipFirehose ? (records.map(r => r.toString("base64")).join('\n') + '\n') : records.join('')
```

Comment:
@czirker Are we expecting the person using this to be gzipping the content themselves?